src/components/Stakeholders/StakeholderDetail/Header.js

```diff
@@ -412,11 +413,18 @@ export default function StakeholderDialogHeader({
                   onKeyDown={handleEnterPress}
                 />
               ) : (
-                <Typography className={classes.fields}>
-                  <Link href={details?.map} target="_blank">
-                    {details?.map}
-                  </Link>
-                </Typography>
+                <>
+                  {details?.map && (
+                    <Typography className={classes.fields}>
+                      <Link
```

Comment:
It would be a good idea to move the `<Link>` outside of the `<Typography>` element while making this change. It probably doesn't matter so much here, but it can cause odd behaviour if the link is too far in.

Reply:
Thanks for the comment, I removed the Typography, and use the props on Typography on Link. Double checked the styles on browser.
![image](https://user-images.githubusercontent.com/10886708/192898703-61ad9cec-b641-4ff9-8801-5d3d89778f62.png)